wasm-runtime/polkadot/src/primitives/tests.rs

```diff
@@ -1,241 +0,0 @@
-// Copyright 2017 Parity Technologies (UK) Ltd.
-// This file is part of Polkadot.
-
-// Polkadot is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-
-// Polkadot is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-
-// You should have received a copy of the GNU General Public License
-// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
-
-//! Tests.
```

Comment:
tests gone? never to return?!

Reply:
these tests can be re-instated, but they actually aren't particularly useful. Until we actually have a specified codec, testing exact serialization of these types is very fragile and prone to breakage every time we touch one of the primitive types.

Reply:
(they have been re-added regardless)